Mar 21 17:04:46 ansible1 sshd[5568]: Accepted publickey for ansible from 192.168.10.10 port 43350 ssh2: RSA SHA256:n9WE6/qxErd4IznBM1aYfitpBCnyUn1Aks3qHHWwzD4
Mar 21 17:04:46 ansible1 systemd[5577]: pam_unix(systemd-user:session): session opened for user ansible by (uid=0)
Mar 21 17:04:46 ansible1 sshd[5568]: pam_unix(sshd:session): session opened for user ansible by (uid=0)
Mar 21 17:04:49 ansible1 sudo[5821]: ansible : TTY=pts/0 ; PWD=/home/ansible ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-mwjaqckqcqdkgbczfyvvzivaoclknzex ; /usr/libexec/platform-python /home/ansible/.ansible/tmp/ansible-tmp-1679385887.3922172-22144-220640412419728/AnsiballZ_setup.py
Mar 21 17:04:49 ansible1 sudo[5821]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Mar 21 17:04:49 ansible1 sudo[5821]: pam_unix(sudo:session): session opened for user root by ansible(uid=0)
Mar 21 17:04:51 ansible1 sudo[5821]: pam_unix(sudo:session): session closed for user root
